src/components/Modals/FiatRamps/views/Overview.tsx

```diff
@@ -132,6 +137,26 @@ export const Overview: React.FC<OverviewProps> = ({
     setShownOnDisplay(shownOnDisplay)
   }, [accountId, accountMetadata, address, wallet])
 
+  const handleIframeClick = useCallback(
+    ({
+      ramp,
+      assetId,
+      address,
+    }: {
+      ramp: SupportedFiatRampConfig
```

Comment:
Given we have:
```typescript
export type SupportedFiatRamp = Record<FiatRamp, SupportedFiatRampConfig>

export const supportedFiatRamps: SupportedFiatRamp = {
```

We should be able to pass a `FiatRamp` instead of the whole ramp and access it here, property accesses are cheap and passing objects around is risky, even though memoization mitigates it